test/celeritas/mat/Material.test.cc

```diff
@@ -157,6 +164,11 @@ TEST_F(MaterialTest, params)
 
 TEST_F(MaterialTest, material_view)
 {
+    if (CELERITAS_UNITS != CELERITAS_UNITS_CGS)
+    {
+        GTEST_SKIP() << "Test needs updating to include units";
```

Comment:
It looks like you already updated part of this test... were you planning on updating the rest?

Reply:
I got lazy at some point... I just wanted enough to show that the code essentially works, and later (next Christmas?) polish everything to a shine.